### ISSUE TITLE
Uri parse exception in WebDevFS.create. Missing Uri scheme.

### ISSUE THREAD

**Issue by zanderso** [flutter/flutter#49374]
Most frequent crash on crash logging in dev/1.14.3. From `flutter run -d chrome`:

```
FormatException: Scheme not starting with alphabetic character (at character 1)
127.0.0.1:46675
^
  | at _Uri._fail | (uri.dart:1625 )
  | at _Uri._makeScheme | (uri.dart:2186 )
  | at new _Uri.notSimple | (uri.dart:1467 )
  | at Uri.parse | (uri.dart:1024 )
  | at WebDevFS.create | (devfs_web.dart:281 )
  | at <asynchronous gap> | (async )
  | at _ExperimentalResidentWebRunner.run | (resident_web_runner.dart:405 )
  | at <asynchronous gap> | (async )
  | at RunCommand.runCommand | (run.dart:555 )
  | at <asynchronous gap> | (async )
  | at FlutterCommand.verifyThenRunCommand | (flutter_command.dart:637 )
  | at _asyncThenWrapperHelper.<anonymous closure> | (async_patch.dart:75 )
  | at _rootRunUnary | (zone.dart:1134 )
  | at _CustomZone.runUnary | (zone.dart:1031 )
  | at _FutureListener.handleValue | (future_impl.dart:140 )
  | at Future._propagateToListeners.handleValueCallback | (future_impl.dart:682 )
  | at Future._propagateToListeners | (future_impl.dart:711 )
  | at Future._completeWithValue | (future_impl.dart:526 )
  | at _AsyncAwaitCompleter.complete | (async_patch.dart:34 )
  | at _completeOnAsyncReturn | (async_patch.dart:293 )
  | at RunCommand.usageValues | (run.dart )
  | at _asyncThenWrapperHelper.<anonymous closure> | (async_patch.dart:75 )
  | at _rootRunUnary | (zone.dart:1134 )
  | at _CustomZone.runUnary | (zone.dart:1031 )
  | at _FutureListener.handleValue | (future_impl.dart:140 )
  | at Future._propagateToListeners.handleValueCallback | (future_impl.dart:682 )
  | at Future._propagateToListeners | (future_impl.dart:711 )
  | at Future._completeWithValue | (future_impl.dart:526 )
  | at Future._asyncComplete.<anonymous closure> | (future_impl.dart:556 )
  | at _rootRun | (zone.dart:1126 )
  | at _CustomZone.run | (zone.dart:1023 )
  | at _CustomZone.runGuarded | (zone.dart:925 )
  | at _CustomZone.bindCallbackGuarded.<anonymous closure> | (zone.dart:965 )
  | at _microtaskLoop | (schedule_microtask.dart:43 )
  | at _startMicrotaskLoop | (schedule_microtask.dart:52 )
  | at _runPendingImmediateCallback | (isolate_patch.dart:118 )
  | at _RawReceivePortImpl._handleMessage | (isolate_patch.dart:169 )
```

@jonahwilliams

**Comment by jonahwilliams** [comment]
Should be fixed via https://github.com/flutter/flutter/pull/49301

**Comment by jonahwilliams** [comment]
Fixed via #49301
